composer.json

```diff
@@ -45,6 +45,8 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "symfony/security-acl": "~2.4|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
I dont know why you do that, I'm not the one, doing this ;D But still: Its very usefull to provide a php.ini precision for unittests, so that everyone has the same environment. Actually you're relying on precision 8 (default php.ini) without knowing it!